content/en/providers/check-in/sp/_index.md

```diff
@@ -482,14 +480,17 @@ usual but includes the `code_verifier` secret generated in the first request.
 Example request:
 
 ```shell
-curl -X POST "https://aai.egi.eu/oidc/token" \
+curl -X POST "${TOKEN_ENDPOINT}" \
 -d "grant_type=authorization_code" \
--d "code=${code}" \
--d "client_id=${client_id}" \
--d "redirect_uri=${redirect_uri}" \
--d "code_verifier=${code_verifier}" | python -m json.tool
+-d "code=${CODE}" \
+-d "client_id=${CLIENT_ID}" \
+-d "redirect_uri=${REDIRECT_URI}" \
+-d "code_verifier=${CODE_VERIFIER}" | python -m json.tool
```

Comment:
```suggestion
  -d "code=${CODE}" \
  -d "client_id=${CLIENT_ID}" \
  -d "redirect_uri=${REDIRECT_URI}" \
  -d "code_verifier=${CODE_VERIFIER}" | python -m json.tool
```

Reply:
Resolved by commit [Add indent to curl data parameter](https://github.com/EGI-Federation/documentation/pull/332/commits/d4c45e1af19edd9a4d4cdaa64e897a218160bc87)